modules/localization/msf/local_tool/map_creation/lossless_map_to_lossy_map.cc

```diff
@@ -48,7 +49,7 @@ MapNodeIndex GetMapIndexFromMapFolder(const std::string& map_folder) {
   if (zone == "south") {
     index.zone_id_ = -index.zone_id_;
   }
-  std::cout << index << std::endl;
+  AINFO << index;
```

Comment:
Use ADEBUG if appropriate. It applies to all.

Reply:
Fixed.